release.sh

```diff
@@ -48,9 +48,9 @@ package() {
 	
 	if [[ $1 == "darwin" ]]; then
 		mv "$ui/Electron.app" "$ui/Sia.app"
-		rm -r "$ui/Sia.app/Contents/Resources/default_app"
-		cp -R ../app/ "$ui/Sia.app/Contents/Resources/app/"
-		mkdir -p "$ui/Sia.app/Contents/Resources/app/Sia"
+		rm -rf "$ui/Sia.app/Contents/Resources/default_app"
+		rsync -av --progress ../ "$ui/Sia.app/Contents/Resources/app/" --exclude "release/" --exclude ".git/"
+		mkdir -p "$ui/Sia.app/Contents/Resources/app/app/Sia"
```

Comment:
@DavidVorick Other electron apps **don't** have all their js, css, etc. in another folder. We do because separated out all the dot-anything files like .jshintrc and .jsdoc plus our documentation from the functional stuff is just neat and convenient.

Changing the names isn't masking the problem, there's just not a problem that the two `app` folders didn't provoke you to imagine. Having two package.jsons is highly unconventional and much larger of a problem. Maintaining an in-house developed wheel reinvention of a packager script is also unconventional and a problem.

Checkout #115, it makes use of electron-packager and is quite a deal less code that has to be version-bumped, debugged, etc.